src/Table/Column/Checkbox.php

```diff
@@ -47,12 +47,15 @@ public function getHeaderCellHtml(Field $field = null, $value = null): string
                 ->addMoreInfo('field', $field);
         }
         $this->table->js(true)->find('.' . $this->class)->checkbox();
+        $this->table->js(true, new JsExpression('atk.gridCheckboxHelper.masterCheckbox();'));
 
-        return parent::getHeaderCellHtml($field);
+        return $this->getTag('head', [['div', ['class' => 'ui master checkbox ' . $this->class], [['input/', ['type' => 'checkbox']]]]], []);
```

Comment:
![image](https://github.com/atk4/ui/assets/2228672/2bda8c65-4d5a-4fa8-982b-4cfaa92a27ba)

the checkbox column should not have unneeded width

Reply:
Probably a css targeting the checkbox column

Reply:
![image](https://github.com/atk4/ui/assets/8080934/dffa792e-98b0-4cd6-9fc5-49d8f922b065)
Fixed

Reply:
used existing collapsing class and fitted checkbox

Reply:
Done.

Reply:
please submit this as separate PR

Reply:
> used existing collapsing class

good finding of correct change, but I am not sure if we should do it, the issue is non-`collapsing` table itself

> and fitted checkbox

👍, merged in https://github.com/atk4/ui/pull/2154